src/html.sortable.js

```diff
@@ -685,6 +692,9 @@ var sortable = function (sortableElements, options) {
       if (!dragging || !_listsConnected(sortableElement, dragging.parentElement)) {
         return
       }
+      if (maxItems && _filter(_getChildren(sortableElement), _data(sortableElement, 'items')).length >= maxItems) {
```

Comment:
Hey, I am fine with leaving it as long as we mention it in the docs. Could you add this to your PR? Thanks.